docs/sdks/web/add-sdk.md

```diff
@@ -222,33 +222,98 @@ import * as SDCBarcode from '@scandit/web-datacapture-barcode';
+Please ensure that the library version of the imported library corresponds to the version of the external Scandit Data Capture library/engine files retrieved via the `libraryLocation` option, 
+either by ensuring the served files are up-to-date or the path/URL specifies a specific version.
 
 In case a common CDN is used (jsDelivr or UNPKG) the library will automatically, internally set up the correct URLs pointing to the files needed for the matching library version.
-It is highly recommended to handle the serving of these files yourself on your website/server, ensuring optimal compression, correct WASM files MIME type, no request redirections, and correct caching headers usage. This will aid in faster loading.
+**It is highly recommended to handle the serving of these files yourself on your website/server, ensuring optimal compression, correct WASM files MIME type, no request redirections, and correct caching headers usage.** 
+This will aid in faster loading.
+
+## Show loading status with default UI
+
+It could take a while the very first time to download the .wasm files. 
+To show some feedback to the user about the loading status you have two options: 
+- use the default UI provided with the SDK 
+- subscribe to the loading status and update your own custom UI. 
+
+Let’s see how to do it with the default UI first:
+
+```ts
+import { configure, DataCaptureView, DataCaptureContext } from "@scandit/web-datacapture-core"
+
+const view = new DataCaptureView();
+
+view.connectToElement(document.getElementById('data-capture-view'));
+view.showProgressBar();
+view.setProgressBarMessage('Loading ...');
+
+await configure({
+	licenseKey: '-- ENTER YOUR SCANDIT LICENSE KEY HERE --',
+	libraryLocation: '/self-hosted-sdc-lib/',
+	moduleLoaders: [idCaptureLoader({ enableVIZDocuments: true })],
+});
+
+view.hideProgressBar();
+
+const context: DataCaptureContext = await DataCaptureContext.create();
+await view.setContext(context);
+```
+
+## Show loading status with custom UI
+
+You can also subscribe for the [loading status](https://docs.scandit.com/data-capture-sdk/web/core/api/web/loading-status.html) of the library
+by simply attaching a listener like this:
+
+```ts
+import { configure, loadingStatus } from "@scandit/web-datacapture-core"
+loadingStatus.subscribe((info) => {
+	// updateUI(info.percentage, info.loadedBytes)
+});
+
+await configure({
+	licenseKey: 'SCANDIT_LICENSE_KEY',
+	libraryLocation: '/self-hosted-sdc-lib/',
+	moduleLoaders: [barcodeCaptureLoader()],
+});
+```
+
+:::note
+We suggest serving the library files with the proper headers `Content-Length` and `Content-Encoding` if any compression is present.
+In case of totally missing information, we show an estimated progress.
+:::
 
 ## Additional Information
 
+### Server Side Rendering and Server Side Generation
```

Comment:
@gpene I see the content of this MR is not either in staging nor in production. did you miss it in some merge conflict resolution?

Reply:
ah my bad I was expecting it in the wrong place nevermind